cmd/acra-server/acra-server.go

```diff
@@ -452,5 +452,5 @@ func main() {
 
 	// on sighup we run callback that stop all listeners (that stop background goroutine of server.Start())
```

Comment:
It seems that this comment line now belongs to line 426 where we install SIGHUP handler.